can_tools/scrapers/official/NH/nh_demographics.py

```diff
@@ -0,0 +1,128 @@
+        # other selections (in this case, the county selection) from the instance, and there doesn't seem
+        # to be a good way to fix/circumvent this. So, we use two TableauScraper
+        # instances to fetch each respective dose type.
+        # This is an issue we've had in the past with the OH and WI demographic scrapers.
+        for variable in [
+            "Total Individuals with at least 1 Dose",
+            "Total Individuals Fully Vaccinated",
+        ]:
+            func = partial(self._get_county, variable)
+            data.extend(pool.map(func, self.counties))
+        return pd.concat(data)
+
+    @staticmethod
+    def _get_county(variable: str, county: str):
+        engine = TableauScraper()
+        engine.loads(
+            url="https://www.nh.gov/t/DHHS/views/VaccineOperationalDashboard/VaccineDashboard"
+        )
+
+        engine.getWorkbook().setParameter("ShowBy", "Race/Ethnicity")
+        engine.getWorkbook().setParameter("Metric", variable)
+
+        raw_data = []
+        worksheet = engine.getWorksheet("Count and Prop: Map (no R/E with town&rphn)")
+        workbook = worksheet.select("CMN + Town + RPHN", county)
+        raw_data.append(
+            workbook.getWorksheet("Count: Bar Chart").data.assign(location_name=county)
+        )
+
+        data = pd.concat(raw_data)
+        return data
+
+    def normalize(self, data) -> pd.DataFrame:
+        demo_replace = {
+            "Hispanic/Latino": "hispanic",
+            "Black or African American": "black",
+            "Other Races": "other",
+        }
+
+        data = (
+            data.rename(columns={"AGG(Metric Value Nh Residents)-alias": "value"})
+            .assign(
+                dt=self._retrieve_dt("US/Eastern"),
+                vintage=self._retrieve_vintage(),
+                race=lambda row: row["Show by Filter-value"]
+                .replace(demo_replace)
+                .str.lower(),
+            )
+            .pipe(
+                self.extract_CMU,
+                cmu=self.variables,
+                var_name="ATTR(Bar Chart Tooltip NH Res)-alias",
+                skip_columns=["race"],
+            )
+        )
+
+        # combine cities of manchester and nashua into hillsborough county
+        cols_to_keep = [
+            "vintage",
+            "dt",
+            "location_name",
+            "category",
+            "measurement",
+            "unit",
+            "age",
+            "race",
+            "ethnicity",
+            "sex",
+            "value",
+        ]
+        data = data.loc[:, cols_to_keep]
+        data["location_name"] = data["location_name"].replace(
+            {
+                "Rest of Hillsborough": "Hillsborough",
+                "Nashua": "Hillsborough",
+                "Manchester": "Hillsborough",
+            }
+        )
+        group = [col for col in cols_to_keep if col != "value"]
+        return data.groupby(group).sum().reset_index()
```

Comment:
Sometime when we're moving at a more leisurely pace it might be nice to create a helper for this location merging thing...

Reply:
yeah, I agree